app/src/main/res/layout/fragment_driving_license.xml

```diff
@@ -0,0 +1,75 @@
+<?xml version="1.0" encoding="utf-8"?>
+<LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    android:orientation="vertical"
+    tools:context=".presentation.authentification.register.verification.DrivingLicenseFragment"
+    tools:ignore="ContentDescription">
+
+    <com.google.android.material.appbar.AppBarLayout
```

Comment:
That's not what I meant when I said to replace the LinearLayout.

Use `[MaterialToolbar](https://material.io/components/app-bars-top/android)`. It contains a lot of things that can ease your implementation.